README.md

```diff
@@ -76,10 +77,37 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals only with the English locale, it may still need
+globalization to format programming language bytes into human understandable
+language and vice-versa in a effective and reasonable way. For example, to
+display something better than "Edited 1 minutes ago".
+
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize provides number formatting and parsing, date and time formatting and
+parsing, currency formatting, message formatting (ICU message format pattern),
+and plural support.
+
+Our design goals are (a) to leverage the official CLDR JSON data, (b) allow
```

Comment:
It might make sense to have the list and then have a short explanation of each point afterward. For example, someone new to globalization isn't going to know why "leverage the official CLDR JSON data" is something they should even care about.